nextcord/utils.py

```diff
@@ -851,6 +854,68 @@ def escape_mentions(text: str) -> str:
     return re.sub(r"@(everyone|here|[!&]?[0-9]{17,20})", "@\u200b\\1", text)
 
 
+def parse_raw_mentions(text: str) -> List[int]:
+    """A helper function that parses mentions from a string as an array of user IDs
+    matched with the syntax of ``<@user_id>``.
+
+    .. note::
+
+        This does not include role or channel mentions. See :func:`parse_raw_role_mentions`
+        and :func:`parse_raw_channel_mentions` for those.
+
+    .. versionadded:: 2.2
+
+    Parameters
+    -----------
+    text: :class:`str`
+        The text to parse mentions from.
+
+    Returns
+    --------
+    List[:class:`int`]
+        A list of user IDs that were mentioned.
+    """
+    return [int(x) for x in re.findall(r"<@!?(\d{15,20})>", text)]
```

Comment:
In an attempt to future-proof this, I think there shouldn't be a maximum character count

Reply:
This is simply relocated from its original location in Message.raw_mentions without logic changes.

21-digit IDs won't be coming until the year 2770 (probably way beyond Discord's existence) so I think this is fine

https://snowsta.mp/?l=en-us&z=1t&f=atx10uuark-lfls

Reply:
Ah yes